recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
 #  git_depth: 1
 
 build:
-  number: 1
+  number: 2
```

Comment:
In `recipe/meta.yaml`, this is not the only change that is required, because if you have a closer look to CI logs or even [this cgal-cpp-feedstock PR](https://github.com/conda-forge/cgal-cpp-feedstock/pull/28) (cgal-cpp is a requirement for gudhi-feedstock), only cgal-cpp >=5.5 is available with boost 1.78.

_(sorry, but I cannot comment/suggest on piece of code you did not change)_
We need to say:
```yaml
...
  host:
    ...
    - cgal-cpp 5.5.*
  run:
    ...
    - cgal-cpp 5.5.*
...
```

And we have to hope that conda build will take cgal-cpp 5.5.1 (it should as it is available), because there is a bug in cgal 5.5.0 that makes gudhi compilation fails.